apps/site/lib/site_web/controllers/schedule/green.ex

```diff
@@ -41,6 +41,18 @@ defmodule SiteWeb.ScheduleController.Green do
 
   def show(conn, _parmas), do: line(conn, [])
 
+  def line_diagram_api(conn, _) do
+    conn
+    |> call_plug(SiteWeb.ScheduleController.HoursOfOperation)
```

Comment:
Do we need to call all these plugs for the green line version of the API? I noticed the one in line_controller doesn't

Reply:
I want to say that I did verify that we do need all of them, but I'll double-check.